src/collection.js

```diff
@@ -104,8 +106,32 @@ export default class Collection {
     });
   }
 
-  use(transfomer) {
-    this._transformers[transfomer.type].push(transfomer);
+  /**
+   * Configures plugins like RemoteTransformers, LocalTransformers, and
+   * IdSchemas. Will be called from `Kinto#collection`
```

Comment:
Ah, I realize this PR doesn't handle idSchema at all, hence my confusion; we should remove any mention of it in the patch though.